docs/USAGE-GUIDE.md

```diff
@@ -1018,6 +1018,10 @@ set the public keys found in the Certificate into **public_key_out**.
 s2n-tls includes support for resuming from cached SSL/TLS session, provided 
 the caller sets (and implements) three callback functions.
 
+### s2n\_config\_set\_session\_cache\_onoff
+
+**s2n_config_set_session_cache_onoff** enables and disables the session cache. In order to use the session cache feature you will need to set a ticket key with **s2n_config_add_ticket_crypto_key** to encrypt session state inside the cache.
```

Comment:
From the description it's a bit not clear what a session cache feature is? Is it a cache when we're in `S2N_CLIENT` mode, so that s2n implicitly caches and resumes sessions?

Reply:
remove "that will be" - all instances
The first pointer will be set to - change to "The first pointer sets "
In order to use the session cache feature you will need - remove "will"

Reply:
I'm going to remove this addition. The session cache doesn't have anything to do with this release, and there's probably more documentation that needs to go into this feature anyways. I'll create an issue for us to deal with this later.